mithril-common/src/crypto_helper/merkle_map.rs

```diff
@@ -34,6 +37,7 @@ pub trait MKMapValue<K: MKMapKey>: Clone + TryInto<MKTreeNode> {
 pub struct MKMap<K: MKMapKey, V: MKMapValue<K>> {
     inner_map_values: BTreeMap<K, V>,
     inner_merkle_tree: MKTree,
+    can_compute_proof_keys: BTreeSet<K>,
```

Comment:
Just a suggestion but IMHO, I don't find the prefix `can` very suitable here. We have both `can_compute_proof`, which returns a `bool` and `can_compute_proof_keys`, which is a `BTreeSet<K>`.

```suggestion
    provable_keys: BTreeSet<K>,
```

Reply:
Done